bin/node/runtime/src/lib.rs

```diff
@@ -515,6 +515,11 @@ parameter_types! {
 		.get(DispatchClass::Normal)
 		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
 		.saturating_sub(BlockExecutionWeight::get());
+	// Solution can occupy 90% of normal block size
+	pub MinerMaxLength: u32 = Perbill::from_rational(9u32, 10) *
+		*RuntimeBlockLength::get()
+		.max
+		.get(DispatchClass::Normal);
```

Comment:
Follow-up issue: We should make the `submit_unsigned` be of class `DispatchClass::Operational` as it is serving an important role in the system and should be treated slightly better.

Reply:
#8511